docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,130 @@
+#. Install the Proctorio extension at https://getproctorio.com/
+#. Go through the opening interstitials and click "Start System Check". You should briefly see an lti launch URL containing "/lti_consumer/" in the URL bar, and then be redirected to the Proctorio Instructions page.
+#. Immediately navigating back and trying to click "Start Exam" still doesn't allow you to start the exam.
+#. Navigating back to the Proctorio Instructions and refreshing on that page causes no issues
+#. Go through the Proctorio Setup for your Webcam, Desktop, etc. Upon setup completion, your webcam should appear to the right of the screen along with some controls. You should land on a page that says "Sending you back to your exam." with a link to "Return to Exam". Clicking this link should send you to another interstitial. (NOTE: Clicking this link is time-bound so make sure to click it fast)
+#. Clicking "Return to Exam" directs you to a page titled "Important" w/ another header saying "Proctored Exam Rules", and a "Start Exam" Button
+#. Clicking "Start Exam" directs you to the exam.
+#. Clicking "End My Exam" directs you to a "Are you sure you want to end your proctored exam?" Interstitial
+#. Clicking "No I'd like to continue working" sends you back to the exam.
+#. Clicking "Yes end my proctored exam" ends the exam and directs you to a "You have submitted this proctored exam for review" page
+#. In the course view, the exam you took is marked as completed (with a green checkmark)
+#. In a separate exam, go through the setup again and, but instead of clicking on "Return to exam" on the "Sending you back to your exam" page, simply wait, You should be sent to the "Important"/"Start Exam" interstitial automatically.
+#. In this separate exam, start the exam, then open devtools using Cmd+Option+I on Mac (F12 on Windows). This should error out the exam and load a "Error with proctored exam" page AND open the "Proctorio Support" page in another tab
+
+
+Exams Dashboard
+---------------
+Expected Behavior: ACS Endpoint calls work, able to review/reset exam attempts
+
+Dashboard UI
+============
+#. Clicking on the "Special Exams" tab loads the *new* version of the exams dashboard.
+#. Clicking on the "Review Dashboard" tab within the exams dashboard should open a new tab to the Proctorio Dashboard for instructors.
+#. Clicking the "Select An Exam" dropdown shows the list of exams for the course.
+#. Filtering by exam name in the exam selection dropdown brings up the expected exams (e.g. querying "M4-M6" shows all the exams in that section)
+#. Selecting a specific exam (i.e. the ones you just took) loads the respective exam attempts for those exams
+#. The exam attempts are in the expected status, e.g. successful attempts are submitted or verified, and the errored attempts are errored.
+#. Filtering by username only shows exam attempts for that username.
+#. The status changes for these exam attempts look the same in the edx-exams admin
+
+Resetting an exam attempt
```

Comment:
these tests could be a good place to cover other effects of a reset too. Why not test the learner's completion and problem is reset as well here? (same comment goes for the below tests)